core/src/main/resources/jenkins/model/Jenkins/login.jelly

```diff
@@ -24,10 +24,11 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <j:if test="${it.setupWizard != null}">
+  <j:choose>
+  <j:when test="${it.setupWizard.useGeneratedPassword()}">
```

Comment:
@jglick it definitely may be null... normal EL behavior is `it.setupWizard` being null short-circuits the expression to false, though... (and it seems to be doing that here, am I missing something)